packages/components/CHANGELOG.md

```diff
@@ -33,6 +34,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 + <p-button-pure>Some label</p-button-pure>
 ```
 
+- Rename prop `state`'s value `neutral` to `info` for `Inline Notification`, `Banner`, `Toast`
```

Comment:
We should still deprecate but still support `neutral`… and map it to the new `info` state for backwards compatibility reasons.